Airports/EGBK/SMR/Regions.txt

```diff
@@ -0,0 +1,212 @@
+; - background
+REGIONNAME Northampton Sywell
+30982 N052.18.40.003 W000.47.50.029
```

Comment:
Do one of the following:
a) Define ```30982``` in the main Colours.txt file, then replace ```30982``` with whatever you call the colour
b) Find a colour already defined and use that (Would rather you do this please!)